### DISCUSSION TITLE
Changement de nom de guilde

### DISCUSSION THREAD

**Discussion by Voisin1234** [Crownicles/Crownicles#2387] [Suggestions fermées]
Bien sûr, voici le texte avec quelques espaces et interlignes ajoutés pour une meilleure lisibilité :

Bonjour, actuellement, il est impossible de changer le nom de sa guilde. Je ne sais pas si au niveau du script c'est possible ou si cela pourrait causer des problèmes, ou si vous ne savez pas comment faire, ou si vous ne voulez pas.

Au cas où cela serait réalisable, voici mon idée : pour pouvoir changer le nom de sa guilde, on pourrait payer le changement, soit admettons avec 50 000💰 ou alors 50💎. On pourrait même éventuellement utiliser 50 000 points de guilde pour payer le changement.

Pourquoi un coût élevé ? Cela serait pour éviter les abus et les changements trop fréquents, par exemple toutes les 30 secondes. 

Pourquoi faire un changement de nom ? Car la vie évolue, et parfois certaines guildes changent de chef et pourraient avoir envie d'adopter un nouveau nom. (Ce n'est pas notre cas, vive la team Océan).

**Comment by Ravenclaw-l** [comment]
Pourquoi ne pas pouvoir changer le nom de la guilde à titre gratuit 1 à 2 fois par an max.

**Comment by voltou91** [comment]
Salut, oui c'est possible et j'avais commencé à dev la feature après la carte blanche de Drapht, mais elle n'est jamais sortie car je n'avais plus le temps. Je pense pas que Drapht y voit d'inconvénient pour cette issue mais elle attendra après la V.5 car on ne peux plus dev de feature qui doit toucher la base de donnée il me semble.

**Comment by BastLast** [comment]
Yo jz ferme car ya deja au moins une autre suggestion identique, merci de ne pas dupliquer ❤️ #1787

**Reply by Voisin1234** [reply]
Je m’en doutais un peu, mais bon. Par contre tu nous mets le lien D’une suggestion que tu as déjà fermé avant avec  la même réponse qu’il y en avait déjà existant. Mais celle-ci date depuis plus de deux ans ok il y a un un UP mais qui date et on ne sait pas si celle-ci sera validé. Tant pis une de plus ce qui passe à la trappe.😉 Bisous quand même.
